lib/LDKeyPair.js

```diff
@@ -26,29 +30,49 @@ class LDKeyPair {
     this.controller = options.controller;
     this.owner = options.owner;
     // this.type is set in subclass constructor
-    // this.publicKey* and this.privateKey* is handled in subclasses
+    // this.publicKey* and this.privateKey* is handled in sub-classes
   }
-
   /**
-   * Returns subclass-appropriate public key material.
+   * @abstract
+   * @interface
+   * @readonly
+   * @description Abstract method that returns the public key.
+   * @example
+   * > ldKeyPair.publicKey;
```

Comment:
the linter wants one and you really do never know someone might not understand that get functions are expressions and are called with out a parenthesis at the end.

Reply:
If we have a linting rule that requires an example, that needs to be removed.  We will not be adding examples to all of our APIs.

Reply:
Removed it. Also learned a lot about the linter and js-doc from this. that plugin is not quite a pro as you might like it to be. still it checks if js-doc annotations are misspelled which helps and requires an @description to be there.

Reply:
@dlongley @davidlehn @JoshDunnDev 

ok so a bit of difference of opinion here so wanted to get some consensus.
The current docs provide examples for es6 class get methods that are read only.

this means this doc string currently have this following example:
```js
> ldKeyPair.publicKey;
publickey
```

Ed25519KeyPair has

```js
> edKeyPair.publicKey;
2S2Q6MkaFJewa
```

the same for goes private keys too:
```js
> rsaKeyPair.privateKey;
3423432423sdasdas3424234dffs
```

@mattcollier actually requested changing one of these example to privateKey().
I think that demonstrates why an example here is important.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get

ES6 Class getters and setters are expressions which do not require () at the end.
So I'm asking for the group consensus do we keep examples that clearly demonstrate the publicKey and privateKey getters can be accessed like properties or do we remove them and risk confusion?

Reply:
For my part, I completely missed that this was a getter.  At any rate,  I don't know why anyone needs an example for essentially accessing the property of a js object (or a function with no arguments).

Reply:
If a `getter` is actually being used, then correct usage is to access it like a property; it should not be called like a function, i.e. getters and setters create magic properties, not functions.

Note: Since this is magic and can yield unexpected behavior, we typically only use `getters` and `setters` when we messed up something in our modeling previously and need to hide some special functionality (that no one would see externally) to correct it. Not sure if that applies here, but that's a separate issue.

Reply:
As for what I think is your second question, I don't think we don't need examples for every single API call. It might be nice, but that's a lot to create and maintain. We're already failing hard enough at generating any docs, so I don't think setting the bar any higher is a great idea.